BasicSteps/TestPlanReference.cs

```diff
@@ -325,6 +325,50 @@ void UpdateStep()
                             step.OnPropertyChanged("");
                         }
                     }
+
+                    if (currentSerializer == null)
+                    {
+                        // if currentSerializer is set, it means that we are loading a previously saved test plan reference.
+                        // Hence these things will be automatically set up.
+                        // otherwise we need to trasfer mixins and dynamic member values.
```

Comment:
It would not. That is because  the serializer plugins has its own way of applying dynamic members and that information is already saved in the test plan.

So in that otherwise the parameters might be added twice.